core/pom.xml

```diff
@@ -81,7 +81,6 @@
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-catalyst_${scala.binary.version}</artifactId>
       <version>${spark.version}</version>
-      <classifier>tests</classifier>
```

Comment:
this will increase the size of the remorph jar by hundreds of megabytes - do we want it now?

Reply:
This is a proposal, or a idea to validate transpiled queries. 
We can discuss I would really like to have it and add block comments on which query potentially fail during validation.